### PR TITLE
Create TipsFragment (do the xml and update the original backend code in advice and tips module)

### PR DESCRIPTION
**Pull request by PeiHui369**
<table>
<tr>
<td>
Urgency
</td>
<td>

⚪️ **Not urgent**

</td>
</tr>
</table>

### CODE REVIEW

File: app/src/main/res/layout/fragment_profile.xml [hunk 1]
```diff
@@ -138,18 +138,18 @@ we can provide you with the best service"
...
 
     <androidx.constraintlayout.widget.Guideline
-        android:id="@+id/guideline20"
+        android:id="@+id/guideline19"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:orientation="vertical"
-        app:layout_constraintGuide_begin="26dp" />
+        android:orientation="horizontal"
+        app:layout_constraintGuide_begin="297dp" />
```
Comment (teojiesern) [inline app/src/main/res/layout/fragment_profile.xml:153]: Why this change?

Comment (PeiHui369) [reply app/src/main/res/layout/fragment_profile.xml:153]: I mispressed and changed the 19&20 when I reference to the guideline
